qosf.org/qc_mentorship.md

```diff
@@ -48,9 +48,9 @@ If you're interested, you can learn more about the [story behind the program](ht
 
 ## Project Highlights
 
-Cohort one (March 2020 to June 2020): [mentorship projects]({{ 'cohort_1' | absolute_url }}).
+Cohort one (March 2020 to June 2020): [mentorship projects]({{ 'mentorship_cohort_1' | absolute_url }}).
 
-Cohort two (September 2020 to January 2021): mentorship projects will be shared on our [Twitter](https://twitter.com/qosfoundation) prior to being published on our website, as the cohort has just finished.
+Cohort two (September 2020 to January 2021): [mentorship projects]({{ 'mentorship_cohort_2' | absolute_url }}).
```

Comment:
Actually they started in October I think...

Reply:
Fixed!